com.sap.cloud.lm.sl.cf.core/src/test/java/com/sap/cloud/lm/sl/cf/core/cf/clients/CloudServiceOperatorTest.java

```diff
@@ -49,15 +46,13 @@ private void prepareRestTemplateFactory() {
 
     private void prepareClient() throws IOException {
         URL controllerUrl = new URL(CONTROLLER_URL);
+        Mockito.reset(client);
```

Comment:
Why reset? Isn't the new Mocked object be enough here?
If no, please refactor the test in order to be.
Further reading: https://softwareengineering.stackexchange.com/questions/188299/is-this-an-appropriate-use-of-mockitos-reset-method